### DISCUSSION TITLE
playactor version check

### DISCUSSION THREAD

**Discussion by LordZork** [dhleong/playactor#26] [Ideas]
It would be great if `version` was added as a [playactor](https://github.com/dhleong/playactor) `<subcommand>` and feature.

**Example:** 
`playactor version`
**Result:** 
`playactor v0.2.2`

––––––––––

It would be even better if it knew (and indicated) if you are running the latest or an outdated [release](https://github.com/dhleong/playactor/releases).

**Result (latest):** 
`playactor v0.2.2 (latest)`

**Result (outdated):** 
`playactor v0.2.1 (outdated)`
`Latest version: v0.2.2`
`To update, run: npm install -g playactor`

––––––––––

It would be best if it also linked the latest version (to easily/quickly view [release](https://github.com/dhleong/playactor/releases) notes before updating).

**Result (outdated w/ latest version link):** 
`playactor v0.2.1 (outdated)`
`Latest version:` [`v0.2.2`](https://github.com/dhleong/playactor/releases/tag/0.2.2)
`To update, run: npm install -g playactor`

**Comment by dhleong** [comment]
Great idea. The usual way to check version would be `playactor --version`. I'm sure I meant to include this so must just be an oversight that it was omitted. Version checking should be doable; I'll consider including it

**Comment by dhleong** [comment]
This has been implemented! Released in https://github.com/dhleong/playactor/releases/tag/0.3.0; see https://github.com/dhleong/playactor/issues/27 for details.

**Comment by LordZork** [comment]
@dhleong, I am late in seeing your comment and responding, but I wanted to thank you for adding this feature.  As always, your efforts with [playactor](https://github.com/dhleong/playactor) are greatly appreciated!
